versions.tf

```diff
@@ -1,5 +1,9 @@
+# ------------------------------------------------------------------------------
+# SET TERRAFORM AND PROVIDER REQUIREMENTS FOR RUNNING THIS MODULE
+# ------------------------------------------------------------------------------
+
 terraform {
-  required_version = ">= 0.12.20, < 0.14"
+  required_version = ">= 0.12.20, < 0.16"
```

Comment:
missing the test against 0.14 in the commit history

Reply:
should be backward compatible? We tested against v0.14 in the other PR which was closed in favor of this one